samples/core/multiple_outputs/multiple_outputs.ipynb

```diff
@@ -22,48 +22,24 @@
     "Install the `kfp` package if you haven't already."
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
-   "outputs": [],
-   "source": [
-    "!python3 -m pip install 'kfp>=0.1.31' --quiet\n"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Setup project info and imports"
-   ]
-  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
     "tags": [
-     "parameters"
+     "skip-in-test"
```

Comment:
Note, cells that should be skipped in a test should have the "skip-in-test" notebook tag.